corehq/apps/userreports/tests/test_view.py

```diff
@@ -133,6 +133,9 @@ def _delete_everything(cls):
     @classmethod
     def tearDownClass(cls):
         cls._delete_everything()
+        # todo: understand why this is necessary. the view call uses the session and the
+        # signal doesn't fire to kill it.
+        Session.remove()
```

Comment:
@snopoke any thoughts on this? I had to add it to avoid having the "still open connections" issue after tests finished. i think the scope of the test view is different from the actual test cases somehow, and the signal doesn't fire. what's odd is that it used to work until this change.